assets/js/blocks/cart-checkout/cart/full-cart/style.scss

```diff
@@ -63,13 +63,37 @@ table.wc-block-cart-items {
 
 .wc-block-cart-item__total {
 	vertical-align: bottom;
+	font-size: 16px;
+	line-height: 19px;
 }
 
 .wc-block-cart-item__full-price {
-	color: $core-grey-light-800;
+	// inline on mobile, so line prices are aligned vertically
+	display: inline-block;
+	margin-right: 0.5em;
+
+	color: $core-grey-dark-400;
 	text-decoration: line-through;
 }
 
+.wc-block-cart-item__line-total {
+	color: $black;
+}
+
+.wc-block-cart-item__discount-badge {
+	// hidden on mobile
+	display: none;
```

Comment:
This could probably be moved inside the `@include breakpoint( "<782px" ) {` media query.

Reply:
Good point - I was defaulting to "mobile first" style, but I see now we have explicit breakpoint sections in the css. Moved these responsive tweaks into the appropriate place in 07d5717fa3599bc53e5cd9cd2eb48ac0c4ba4b2f